note.py

```diff
@@ -159,27 +170,55 @@ def __note_filename(self, name):
         return os.path.join(self.__notespath, name, "note.md")
 
     def geometry(self, geometry=None):
-        """Returns the geometry (size) of the application window"""
+        """
+        Returns and optionally sets the geometry (size)
+        of the application window.
+
+        :param geometry: Optional geometry of application window (Default: None).
+        :type  geometry: str | None
+
+        :return: Configured geometry (size) of the application window.
```

Comment:
```suggestion
        :return: Configured geometry (of the application window.
```
if position is changed once, it is saved, too

Reply:
fixed